pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -158,6 +159,33 @@ func makeBookiePodSpec(bk *v1alpha1.BookkeeperCluster) *corev1.PodSpec {
 		indexSubPath = IndexDiskName
 	}
 
+	if _, ok = bk.Spec.Options["extraVolumeMounts"]; ok {
+		extraVolumeMounts = strings.Split(bk.Spec.Options["extraVolumeMounts"], ",")
+	}
+
+	var volumes []corev1.Volume
+	if len(extraVolumeMounts) > 1 {
+		for _, vm := range extraVolumeMounts {
+			s := strings.Split(vm, "=")
+			v := corev1.Volume{
+				Name: s[0],
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
```

Comment:
Is that possible to specify the Volumesource such as emptyDir here?

Reply:
I added an additional option to support emptyDirs: emptyDirVolumeMounts. It is implemented in a manner similar to hostPathVolumeMounts: emptyDirVolumeMounts: "baz=/tmp/baz,quux=/tmp/quux".

Reply:
Thanks!